be/src/util/string_util.h

```diff
@@ -57,6 +60,23 @@ struct StringCaseLess {
     }
 };
 
+struct PathHash {
```

Comment:
why do you define a struct? A function is better.

Reply:
I don't know why other 'functions' in string_util are defined as struct. Just for unification.

Reply:
em...
Other struct can be used for `std::map` or `std::unordered_map`. However your `struct` can't be used as a comparator or a hasher for these container. 

So, you'd better to define a function here.

Reply:
I got it. Fixed.